### DISCUSSION TITLE
SEB Not Responding

### DISCUSSION THREAD

**Discussion by Celeste2011** [SafeExamBrowser/seb-win-refactoring#111] [Q&A]
I am using Mettl Secure Browser and SEB for my assessments. I regularly uninstall and update the software prior to every assessment to ensure that it works, and when I did the same today I am unable to access the test window. The SEB application is stuck in the loading screen and shows the status "Not Responding" in the Task Manager. I am having to restart my laptop every time, but it doesn't fix the issue. I have followed the SEB Resetter manual and successfully completed the steps, but this issue still persists.

However, when I did a fresh installation of the same setup on a friend's laptop, there were no issues. I have confirmed this by restarting the laptop multiple times.

SEB version is 2.4.1, using Windows 10.

**Comment by dbuechel** [comment]
SEB 2.x is not actively developed anymore, we highly recommend switching to version 3.x. You might find an indication about the underlying issue by checking the application log of SEB 2.x, to be found under `%AppData%\SafeExamBrowser`
